### PR TITLE
NIFI-13089: Updating PR Template and README for new UI

### PR DESCRIPTION
**Pull request by mcgilman**
NIFI-13089:
- Updating PR Template and README for new UI.

### CODE REVIEW

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/README.md [hunk 1]
```diff
@@ -1,41 +1,73 @@
...
+            -   ui
+        -   settings
+            -   feature
+            -   service
+            -   state
+            -   ui
+        -   ...
+    -   service
+    -   state
+    -   ui
```
Comment (rfellows) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/README.md:40]:
```suggestion
app
├── pages
│   ├── flow-designer
│   │   ├── feature
│   │   ├── service
│   │   ├── state
│   │   └── ui
│   ├── settings
│   │   ├── feature
│   │   ├── service
│   │   ├── state
│   │   └── ui
├── service
├── state
└── ui
```

**Review by rfellows — COMMENTED** [review]
Just one minor improvement suggested. Otherwise this is good.

**Comment by mcgilman** [comment]
> @mcgilman For awareness, PR #8677 includes moving the `nifi-framework` module, which would impact the location of the source links listed in the new `UI Contributions` section of the template. Perhaps simply highlighting `nifi-web-ui` and `nifi-web-frontend` without linking would be sufficient?

Thanks for the comment. I am aware of #8677. I will update once everything is in. I think the link is helpful as the `new UI` link takes the user directly to the location of the README. This should help folks that have opened PRs updating the existing UI get an introduction to the structure and layout in the new UI.

**Review by rfellows — APPROVED** [review]
Thanks @mcgilman 👍

**Comment by exceptionfactory** [comment]
> > @mcgilman For awareness, PR #8677 includes moving the `nifi-framework` module, which would impact the location of the source links listed in the new `UI Contributions` section of the template. Perhaps simply highlighting `nifi-web-ui` and `nifi-web-frontend` without linking would be sufficient?
> 
> Thanks for the comment. I am aware of #8677. I will update once everything is in. I think the link is helpful as the `new UI` link takes the user directly to the location of the README. This should help folks that have opened PRs updating the existing UI get an introduction to the structure and layout in the new UI.

Sounds good, thanks!

**Pull request merged**